test/e2e/k8s_util.go

```diff
@@ -242,6 +242,37 @@ func (k *KubernetesUtils) CreateOrUpdateNetworkPolicy(ns string, netpol *v1net.N
 	return np, err
 }
 
+// DeleteTier is a convenience function for deleting an Antrea Policy Tier with specific name.
+func (k *KubernetesUtils) DeleteTier(name string) error {
+	_, err := k.securityClient.Tiers().Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrapf(err, "unable to get tier %s", name)
+	}
+	log.Infof("deleting tier %s", name)
+	if err = k.securityClient.Tiers().Delete(context.TODO(), name, metav1.DeleteOptions{}); err != nil {
+		return errors.Wrapf(err, "unable to delete tier %s", name)
+	}
+	return nil
+}
+
+// CreateTier is a convenience function for creating an Antrea Policy Tier by name and priority.
+func (k *KubernetesUtils) CreateNewTier(name string, tierPriority int32) (*secv1alpha1.Tier, error) {
+	log.Infof("creating tier %s", name)
+	_, err := k.securityClient.Tiers().Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		tr := &secv1alpha1.Tier{
```

Comment:
nit: normally "error" is handled first and indented so that the normal code path that creates the tier can be kept at a minimum indention and be scanned quickly: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow, though the "error" path is "err == nil" in this case.

Reply:
probably should pick a different variable name than 'err' here. Will address this in a follow-up patch.

Reply:
I'm fine with keeping "err", but a comment explaining "err == nil" might be helpful.

Reply:
perhaps it can skip getting the tier first because Create will return AlreadyExist anyway.

Reply:
Sure. Thanks for the suggestion.